subsys/bluetooth/controller/ll_sw/ull_conn.c

```diff
@@ -82,9 +82,9 @@
 LOG_MODULE_REGISTER(bt_ctlr_ull_conn);
 
 static int init_reset(void);
-#if !defined(CONFIG_BT_CTLR_LOW_LAT_ULL)
+#if !defined(CONFIG_BT_CTLR_LOW_LAT)
```

Comment:
I'm not familiar with the Kconfig option, but why is it being modified from `CONFIG_BT_CTLR_LOW_LAT_ULL` to `CONFIG_BT_CTLR_LOW_LAT` in an "`ull`" file? Seems to me that `CONFIG_BT_CTLR_LOW_LAT_ULL` is more appropriate here than `CONFIG_BT_CTLR_LOW_LAT`

Reply:
`CONFIG_BT_CTLR_LOW_LAT` is alternative design of the Controller (like for nRF51 but usable for nRF52x also), and `CONFIG_BT_CTLR_LOW_LAT_ULL` is a low latency execution of ULL processing that yields instead of looping in `while`.

the `depends on BT_CTLR_LOW_LAT`  can be removed in separate PR after I can add proper tests. Currently `BT_CTLR_LOW_LAT` is covered in some bsim mesh tests, so that mesh demo sample can continue to work on bbc:microbit boards (nrf51 based ones).

https://github.com/zephyrproject-rtos/zephyr/blob/9b3e8ddac3c8325e9cfa8e121b599d2559d6962c/subsys/bluetooth/controller/Kconfig.ll_sw_split#L877-L895